src/main/java/io/javalin/Javalin.java

```diff
@@ -13,33 +13,26 @@
 import io.javalin.core.JavalinConfig;
 import io.javalin.core.JavalinServer;
 import io.javalin.core.JettyUtil;
-import io.javalin.core.event.EventListener;
-import io.javalin.core.event.EventManager;
-import io.javalin.core.event.HandlerMetaInfo;
-import io.javalin.core.event.JavalinEvent;
-import io.javalin.core.event.WsHandlerMetaInfo;
+import io.javalin.core.event.*;
```

Comment:
It looks like our editors are using different import settings. These are my settings: 
![image](https://user-images.githubusercontent.com/1521451/57567251-b21dbe00-73d6-11e9-9623-4fee4a21d66f.png)

Intellij settings export: 
```
<code_scheme name="tipsy-format" version="173">
  <option name="CLASS_COUNT_TO_USE_IMPORT_ON_DEMAND" value="99" />
  <option name="NAMES_COUNT_TO_USE_IMPORT_ON_DEMAND" value="99" />
  <option name="PACKAGES_TO_USE_IMPORT_ON_DEMAND">
    <value />
  </option>
  <option name="IMPORT_LAYOUT_TABLE">
    <value>
      <package name="" withSubpackages="true" static="false" />
      <package name="" withSubpackages="true" static="true" />
    </value>
  </option>
  <option name="RIGHT_MARGIN" value="200" />
  <option name="FORMATTER_TAGS_ENABLED" value="true" />
  <HTMLCodeStyleSettings>
    <option name="HTML_DO_NOT_INDENT_CHILDREN_OF" value="" />
  </HTMLCodeStyleSettings>
  <JavaCodeStyleSettings>
    <option name="CLASS_COUNT_TO_USE_IMPORT_ON_DEMAND" value="99" />
    <option name="NAMES_COUNT_TO_USE_IMPORT_ON_DEMAND" value="99" />
    <option name="PACKAGES_TO_USE_IMPORT_ON_DEMAND">
      <value />
    </option>
    <option name="IMPORT_LAYOUT_TABLE">
      <value>
        <package name="" withSubpackages="true" static="false" />
        <package name="" withSubpackages="true" static="true" />
      </value>
    </option>
  </JavaCodeStyleSettings>
  <codeStyleSettings language="JAVA">
    <indentOptions>
      <option name="CONTINUATION_INDENT_SIZE" value="4" />
    </indentOptions>
  </codeStyleSettings>
  <codeStyleSettings language="Twig">
    <indentOptions>
      <option name="CONTINUATION_INDENT_SIZE" value="4" />
    </indentOptions>
  </codeStyleSettings>
</code_scheme>
```